src/pages/user/Settings.js

```diff
@@ -1,8 +1,19 @@
 import { Helmet } from 'react-helmet';
 import React, { Component } from 'react';
 import { Container, Row, Col, Form, InputGroup, Button } from 'react-bootstrap';
+import { actions as themeActions } from 'reducers/theme';
+import { getThemeName } from 'selectors/theme';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import ToggleButton from 'components/ToggleButton/ToggleButton';
+import PropTypes from 'prop-types';
+
+export class UserSettings extends Component {
+  static propTypes = {
+    theme: PropTypes.string,
```

Comment:
Do you mind sorting the props alphabetically, at least for the propTypes declaration?